aiounifi/models/device.py

```diff
@@ -722,6 +723,49 @@ def create(
         )
 
 
+@dataclass
+class DeviceSetLedStatus(ApiRequest):
+    """Request object for setting LED status of device."""
+
+    @classmethod
+    def create(
+        cls,
+        device: "Device",
+        status: str = "on",
+        brightness: int | None = None,
+        color: str | None = None,
+    ) -> Self:
+        """Set LED status of device."""
+
+        data = {"led_override": status}
```

Comment:
```suggestion
        data: dict[str, int | str] = {"led_override": status}
```
You also put integers into data so you need to specify the possible types